tex/gregoriotex-spaces.tex

```diff
@@ -1484,58 +1495,33 @@
 }
 
 % The common internals for \grecreatedim and \grechangedim.
```

Comment:
Not your fault (probably mine), but can you fix the typo in this comment and add the `@` that should be in `\gre@createdim`?

Reply:
Done